### PR TITLE
FCREPO-1989: Minor cleanup of compiler warnings

### PR DESCRIPTION
**Pull request by ajs6f**
https://jira.duraspace.org/browse/FCREPO-1989

### CODE REVIEW

File: fcrepo-http-api/pom.xml [hunk 1]
```diff
@@ -77,11 +77,6 @@
       <version>${project.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.fcrepo</groupId>
-      <artifactId>fcrepo-connector-file</artifactId>
-      <version>${project.version}</version>
```
Comment (acoburn) [inline fcrepo-http-api/pom.xml]: This can't be entirely removed, but you can scope it to `test`.

Comment (ajs6f) [inline fcrepo-http-api/pom.xml]: I can do that, but I want to ask now-- why do we need this for a test? This seems like a bad and confusing dependency.

Comment (acoburn) [inline fcrepo-http-api/pom.xml]: It seems like moving the projection tests into `fcrepo-webapp` might make the most sense.

Comment (ajs6f) [inline fcrepo-http-api/pom.xml]: It would make _more_ sense. I guess it depends on what view we take of the projection functionality. @awoods, thoughts?

**Comment by awoods** [comment]
It seems most appropriate to move projection outside of core, into fcrepo-webapp-plus... either now or soon.

BTW, this PR should be targeted to the `DEV` branch instead of `master`.

**Comment by acoburn** [comment]
@awoods If I understand you correctly, you're suggesting that the entire `fcrepo-connector-file` module be moved into the `fcrepo4-exts` organization. If that's the case, I'm _definitely_ in favor of that.

**Comment by awoods** [comment]
That is indeed what I am saying... but that probably deserves its own ticket.

**Comment by acoburn** [comment]
Yes, that's certainly beyond the scope of "Minor cleanup of compiler warnings"

**Comment by ajs6f** [comment]
:+1:  to moving the connector jazz out of the core. Meanwhile, I'm just going to shift the scope for the dep here.

**Comment by ajs6f** [comment]
https://jira.duraspace.org/browse/FCREPO-1992

**Comment by awoods** [comment]
Note, this PR needs to be re-targeted to the `DEV` branch instead of `master`.

**Comment by ajs6f** [comment]
Closed in favor of https://github.com/fcrepo4/fcrepo4/pull/1023

**Pull request closed without merging**